src/SprykerEco/Yves/Adyen/Theme/default/views/klarna-invoice/klarna-invoice.twig

```diff
@@ -0,0 +1,12 @@
+{% extends model('template') %}
+
+{% define data = {
+    form: form
+
+} %}
+
+{% block body %}
+    <div class="col col--sm-12 col--md-6">
+        {{ form_widget(data.form.adyenKlarnaInvoice) }}
```

Comment:
But do we really need templates for it? It's just regular form field rendering.

Reply:
I think the question for @aleksey-kotsuba